### PR TITLE
Fixed #499

### LINKED ISSUE #499: Fix Paper Size in Analysis Model
image

### PR DESCRIPTION
**Pull request by Domily11**
Paper now expands when changed to analysis view

### CODE REVIEW

**Comment by amgrubb** [comment]
@Domily11 

- [x] There is a bug with the relationship inspector, I think you need to make another update. When navigating back from the analysis view the relationship inspector doesn't look correct.

**Comment by Domily11** [comment]
> @Domily11
> 
> * [x]  There is a bug with the relationship inspector, I think you need to make another update. When navigating back from the analysis view the relationship inspector doesn't look correct.

Yep, it now works correctly I believe

File: leaf-ui/js/onFunctions.js [hunk 1]
```diff
@@ -433,6 +433,9 @@ paper.on("link:options", function (cell) {
 
         // Remove model only elements 
         $('.model-only').css("display", "none");
+        $('.inspector').css("display", "none");
+        $('#paper').css("right", "0px");
+
```
Comment (Manarnz) [inline leaf-ui/js/onFunctions.js:438]: Maybe delete this additional line (I'm trying to be like Ali ahah).

**Review by Manarnz — COMMENTED** [review]
You did a great job Cat! Nice solution to the bug.

**Review by amgrubb — APPROVED** [review]
Looks great! Please `Squash and merge`.

**Pull request merged**